app/index.js

```diff
@@ -148,10 +155,14 @@ app.on('ready', () =>
 
         const hwin = new Window({width, height, x: startX, y: startY}, cfg, fn);
         windowSet.add(hwin);
+        if (mainWindowState.maximized) {
+          hwin.maximize();
```

Comment:
The maximize/restore button state remains the same, though everything else works fine. the maximize button state is defined in header.js. Could use some help over here fixing the maximize button not showing the maximized window state.

```
const maxButtonHref = this.props.maximized
      ? './renderer/assets/icons.svg#restore-window'
      : './renderer/assets/icons.svg#maximize-window';

```
```
handleMaximizeClick() {
    if (this.props.maximized) {
      this.props.unmaximize();
    } else {
      this.props.maximize();
    }
  }
```